students/pkoleyni/Lesson03/Slicing_Lab.py

```diff
@@ -0,0 +1,66 @@
+def exchange_first_last(seq):
+    """
+    :param seq: any python seq, like String,List,Tuple
+    :return:
+    """
+    first = seq[0]
+    last = seq[-1]
+    middle = seq[1:-1]
+    if type(seq) == str:
+        new_seq = last + middle + first
+        return new_seq
+    elif type(seq) == list:
+        middle.insert(0, last)
+        middle.append(first)
+        return middle
+
+def every_other_item(seq):
+    new_list=[]
+    new_string = ''
+    for i in range(0,len(seq),2):
+        new_list.append(seq[i])
+    if type(seq) == list:
+        return new_list
+    elif type(seq) == str:
+        return new_string.join(new_list)
```

Comment:
the above two functions are the only ones that have issues, below functions look fine!

the issue with above function is you're not using slicing exclusively - you are mixing indexing and loops as well.

Try to use slicing only (hint: this should be only one line of code) - if you still get stuck comment here and I will help out!